pkg/plugin/conftest/plugin.go

```diff
@@ -210,7 +214,10 @@ func (p *plugin) ParseConfigAuditReportData(logsReader io.ReadCloser) (v1alpha1.
 	var successesCount, warningCount, dangerCount int
 
 	for _, cr := range checkResults {
-		successesCount += cr.Successes
+		// Conftest reportedly returns negative count of passed tests is some cases: https://github.com/open-policy-agent/conftest/issues/464
+		if cr.Successes > 0 {
```

Comment:
This is the actual fix. The rest is adding unit tests.

Reply:
I read through the conftest bug report and to me it sounds like this happens if there are more failures than passes in a rego file. If we add this check of >0 will we lose anything?

Reply:
I believe that we are not loosing anything because Conftest does not return successful checks anyway, only warnings, failures and successes count. Successes count as as a negative integer is just confusing.